### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by anderCM**
- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable /components.
- Add [React Router](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router)
- Add two <Route>s to your application:
  - /books - the default view:
    - Should display a list of books
    - Should display a form for adding a book (no functionality yet).
  - /categories:
    - Should display a button "Check status" only.
- Create a <Navbar> component that includes <Link>s to the two routes you created
- Create a <Book> component that displays a single book (props are title and author.
  - Each Book has a "remove" button (no functionality yet)
- Create a <Form> component (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).

### CODE REVIEW

File: src/Routes/Books/Books.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <div className="container border rounded my-3 p-5">
+        {books.map((book) => (
+          <div className="card mb-3" key={book.id}>
+            <div className="card-body">
+              <h6 className="card-title">{book.genre}</h6>
+              <h4 className="card-text">{book.title}</h4>
+              <p className="text-muted">{book.author}</p>
+              <ActionsBook />
+            </div>
+          </div>
```
Comment (amejid) [inline src/Routes/Books/Books.jsx]: - [x] Nice job utilizing reusable components. :+1: Let's work on some minor details to achieve the project requirements. Kindly extract these lines into their own component and pass the data as `props` as suggested in the requirements. ✍️

**Project Requirements**
Screenshot from 2023-02-24 10-33-43

File: src/Routes/Categories/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>Categories</div>
```
Comment (amejid) [inline src/Routes/Categories/Categories.jsx]: - [x] Kindly put a button with the text `Check status` as stated in the project requirements. 😎️

**Project Requirements**
Screenshot from 2023-02-24 10-34-00

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @anderCM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Gitflow :heavy_check_mark: 
- React router correctly set up :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @anderCM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Addressed change requested


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Routes/Books/Books.jsx (hunk 1 above)
Comment (anderCM) [reply src/Routes/Books/Books.jsx]: Thanks for adding this information

File: src/Routes/Categories/Categories.jsx (hunk 2 above)
Comment (anderCM) [reply src/Routes/Categories/Categories.jsx]: Sure, I will add the required text. Thanks for adding this information

**Pull request merged**